packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -1136,6 +1138,17 @@ const TagBox = (SelectBox as any).inherit({
     }
   },
 
+  _setTagAria(element, value) {
```

Comment:
I prefer we make names more precise
```suggestion
  _setTagAria($tag, tagText) {
```

Reply:
[d7c9582b5f642cd4f11e3d8418ac1a59ca7ddb17]